discovery/triton/triton_test.go

```diff
@@ -112,6 +136,7 @@ func TestTritonSDRefreshMultipleTargets(t *testing.T) {
 	var (
 		dstr = `{"containers":[
 		 	{
+                                "groups":["foo","bar","baz"],
```

Comment:
indentation is off

Reply:
This appears to "look" incorrect because of the way GitHub formats it. I double checked the source code, re-ran `go fmt`, and also ran `make style` without issue.